plugins/source/aws/client/client.go

```diff
@@ -116,12 +117,16 @@ func (c *Client) Logger() *zerolog.Logger {
 }
 
 func (c *Client) ID() string {
-	return strings.TrimRight(strings.Join([]string{
+	idStrings := []string{
 		c.AccountID,
 		c.Region,
 		c.AutoscalingNamespace,
 		string(c.WAFScope),
-	}, ":"), ":")
+	}
+	if c.LanguageCode != "" {
```

Comment:
Since we do a TrimRight(":") two lines below, we don't need this block. Should just add `c.LanguageCode` to the `[]string` defn above.